Content.Shared/_RMC14/Xenonids/Construction/SharedXenoConstructionSystem.cs

```diff
@@ -384,31 +400,19 @@ private void OnHiveConstructionNodeAddPlasmaDoAfter(Entity<XenoConstructionCompo
         if (_net.IsClient)
             return;
 
-        var hive = CompOrNull<HiveMemberComponent>(target)?.Hive;
-        var spawn = Spawn(node.Spawn, transform.Coordinates);
-        var member = EnsureComp<HiveMemberComponent>(spawn);
-        _hive.SetHive((spawn, member), hive);
-
-        _adminLogs.Add(LogType.RMCXenoOrderConstructionComplete, $"Xeno {ToPrettyString(xeno):xeno} completed construction of {ToPrettyString(target):xeno} which turned into {ToPrettyString(spawn):spawn} at {transform.Coordinates}");
+        // Don't need to check if the construction is limited here since it's checked when spawning the "ghost"
```

Comment:
Is there a case where two can be built if two xenos place the ghost down on the same tick?
Might be useful to pull the check out into its own method that is also checked here

Reply:
starting and ending the ghost doafter both check CanOrderConstructionPopup, so if 2 xenos tried to build ghosts in the same tick an arbitrary one (entity spawn order prob) would succeed and the other fails

should i still add a check anyway to the plasma bit?